api/s2n.h

```diff
@@ -1116,6 +1116,9 @@ extern int s2n_config_set_session_tickets_onoff(struct s2n_config *config, uint8
 
 /**
  * Enable or disable session caching.
+ * 
+ * @note Session caching will not be turned on unless all three session cache callbacks are set
+ * prior to calling this function.
```

Comment:
Can we open an issue to fix this, if there isn't one already an open issue? It seems like a sharp edge.

Reply:
I could see a case where this is the desired behavior, so caching is not attempted with no caching callbacks, but it is definitely bizarre. I made an issue.